homeassistant/components/scrape/sensor.py

```diff
@@ -210,21 +215,22 @@ def _async_update_from_rest_data(self) -> None:
             SensorDeviceClass.TIMESTAMP,
         }:
             self._attr_native_value = value
+            self._attr_available = self._available
             self._process_manual_data(raw_value)
             return
 
         self._attr_native_value = async_parse_date_datetime(
             value, self.entity_id, self.device_class
         )
+        self._attr_available = self._available
         self._process_manual_data(raw_value)
-        self.async_write_ha_state()
```

Comment:
I wasn't expecting this line to be removed. Please could you help explain the reason for this?

Reply:
It's handled by the super in `_handle_coordinator_update()`. No need to write state twice.